pom.xml

```diff
@@ -194,12 +189,6 @@
   </dependencyManagement>
   <dependencies>
     <!-- static analysis -->
-    <dependency>
-      <groupId>com.google.code.findbugs</groupId>
-      <artifactId>annotations</artifactId>
-      <scope>provided</scope>
-      <optional>true</optional>
```

Comment:
so this was ickily added to prevent findbugs being included in the HPI as some libraries do not declare it optional.
this will I beleive mean that findbugs annotations will slip back into many HPIs.

Reply:
it's still there but under the profile now.

Reply:
it is only in the profile for `findbugs`.  it should be there for `spotbugs` too.

Reply:
it's definite there too: https://github.com/casz/plugin-pom/blob/spotbugs/pom.xml#L1455-L1472